packages/xrpl/test/integration/transactions/escrowCancel.test.ts

```diff
@@ -1,72 +1,136 @@
 import { assert } from 'chai'
-import _ from 'lodash'
-import { EscrowCancel, EscrowCreate } from 'xrpl-local'
 
+import { EscrowCancel, EscrowCreate } from '../../../src'
 import serverUrl from '../serverUrl'
-import { setupClient, teardownClient } from '../setup'
-import { generateFundedWallet, getXRPBalance, testTransaction } from '../utils'
+import {
+  setupClient,
+  teardownClient,
+  type XrplIntegrationTestContext,
+} from '../setup'
+import {
+  // calculateWaitTimeForTransaction,
+  generateFundedWallet,
+  // getXRPBalance,
+  testTransaction,
+  runCommand,
+} from '../utils'
+
+// NOTE: Because ledger accept is called among multiple tests, the actual ledger close time is not
```

Comment:
Please add a followup to fix these tests

Reply:
Done